deploy/complete/kubernetes/mushop.yaml

```diff
@@ -1263,6 +1264,48 @@ spec:
                 secretKeyRef:
                   name: orders-oadb-connection
                   key: oadb_service
+                  
+            - name: OCI_TENANT_ID
```

Comment:
External entries like OCI credentials and specific entries for a tenancy should never be mandatory.

Reply:
Hmm, agree. I would suggest mushop-monit-scenario.yaml